docs/component-adapter/component-adapter-faq.md

```diff
@@ -1,3 +1,12 @@
+---
+title: Component Adapter FAQ
+category: 6849ddd92905ee0053320687
+slug: component-adapter-faq
+hidden: false
+parentDoc: 6852ed81d85ebd00247da5a0
```

Comment:
Is there a way we can do this without storing the category and parent doc in the frontmatter? Worried this is going to be hard to add new documents. Maybe we could chat more about this tomorrow

the hope would be that we could use a unique identifier (file path + filename) and then store identifiers like this in the lockfile instead. I may be missing something tho

Reply:
Same concern. FWIW see docs https://docs.readme.com/main/docs/rdme (Link looks generic is not, its for the CLI)

Reply:
Just pushed a change that creates a temporary folder to mirror our docs. We add the parentDoc ID and category ID there, and that folder becomes the source for our GitHub Action to publish from.

It makes me wonder if we should have front matter at all. However we have options for `order` `description` `hidden` which seem like a smart place for them.